requirements.txt

```diff
@@ -4,10 +4,11 @@ https://github.com/wazo-platform/wazo-auth-client/archive/master.zip
 https://github.com/wazo-platform/xivo-fetchfw/archive/master.zip
 https://github.com/wazo-platform/xivo-lib-python/archive/master.zip
 jinja2==2.10
+markupsafe==1.1.0  # from jinja2
 pyopenssl==19.0.0
 pyyaml==3.13
 requests==2.21.0
 six==1.12.0
 stevedore==1.29.0
 twisted==18.9.0
-zope.interface==4.3.2
+zope.interface==4.7.2
```

Comment:
Is this version required? It's not available in Debian at the moment

Reply:
For this PR I think we can avoid changing the version, but I think I was not able to make it work using Python3 with version 4.3.2

Reply:
The changelog for zope says `4.6.0` "Adds support for Python 3.7`. So, I would assume we need at least that.
But debian does ship with zope 4.3.2 and Python 3.7.... which is confusing.